_summaries/sukhbaatar2015end-to-end_2.md

```diff
@@ -1,19 +1,42 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{sukhbaatar2015end-to-end}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: mmcinnestaylor # Your GitHub id
+score: 8 # How did you like this paper 0(dislike) to 10(love)
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Core Idea
+* End-To-End Memory Networks are a type of Memory Network with a possibly large external memory which is are trained end-to-end.
+* End-to-end training requires less supervision, thus improving applicability in real world settings.
+
+## Technical Implementation
+![Architecture](sukhbaatar2015end-to-end_2_b.png) 
```

Comment:
I would put this after the 2nd image (the one about the example input / query / output etc.) just to make the summary flow a bit better.  

In my opinion, start with "What is this model actually solving", because the paper makes so much more sense if you start with that perspective